sdk/peering/Azure.ResourceManager.Peering/api/Azure.ResourceManager.Peering.netstandard2.0.cs

```diff
@@ -944,4 +946,11 @@ public partial class PeeringSku
         public static bool operator !=(Azure.ResourceManager.Peering.Models.PeeringTier left, Azure.ResourceManager.Peering.Models.PeeringTier right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class RpUnbilledPrefix
```

Comment:
We need to add this word `Unbilled` to the cspell file here: https://github.com/Azure/azure-sdk-for-net/blob/main/.vscode/cspell.json#L7

Reply:
added unbilled to the dictionary